src/main/java/com/ohmyclass/api/components/user/controller/IUserController.java

```diff
@@ -6,30 +6,30 @@
 import com.ohmyclass.api.util.ApiConst;
 import com.ohmyclass.api.util.communication.Response;
 import com.ohmyclass.util.other.Development;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping(ApiConst.URL_ACCOUNT)
 public interface IUserController {
 
-	@PostMapping(ApiConst.GET)
+	@PostMapping(ApiConst.LOGIN)
 	Response<UserOutDTO> login(UserInDTO user);
 
-	@PostMapping(ApiConst.GET)
+	@PutMapping(ApiConst.REGISTER)
 	Response<UserOutDTO> register(UserInDTO user);
 
 	@Development
 	@PostMapping(ApiConst.GET)
```

Comment:
Because it's a dev only method. It's not that important and will be removed for the final release anyway.
The route acts as a URL, the annotation as a protocol. @GetMapping won't just magically create a "/get".